backend/btrixcloud/subs.py

```diff
@@ -182,6 +184,51 @@ async def cancel_subscription(self, cancel: SubscriptionCancel) -> dict[str, boo
         await self.add_sub_event("cancel", cancel, org.id)
         return {"canceled": True, "deleted": deleted}
 
+    async def send_trial_end_reminder(
+        self,
+        reminder: SubscriptionTrialEndReminder,
+    ):
+        """Send a trial end reminder email to the organization admins"""
+
+        org = await self.org_ops.find_org_by_subscription_id(reminder.subId)
+
+        if not org:
+            print(f"Organization not found for subscription ID {reminder.subId}")
+            raise HTTPException(
+                status_code=404, detail="org_for_subscription_not_found"
+            )
+
+        if not org.subscription:
+            print(
+                f"Subscription not found for organization ID {org.id} with sub id {reminder.subId}"
+            )
+            raise HTTPException(status_code=500, detail="subscription_not_found")
+
```

Comment:
```suggestion
```

Since we just fetched the org by its `subscription.subId` value equalling `reminder.subId` above, probably safe to assume the subscription is set on the org here.

Reply:
Ah, fair enough. Feel free to keep and just swap to a `400` if you'd prefer :)